### ISSUE TITLE
Failed at the libxmljs@0.8.1 install script: error code ELIFECYCLE

### ISSUE THREAD

**Issue by nemobis** [libxmljs/libxmljs#222]
303 info postuninstall libxmljs@0.8.1
304 error libxmljs@0.8.1 install: `node-gyp rebuild`
304 error Exit status 1
305 error Failed at the libxmljs@0.8.1 install script.
305 error This is most likely a problem with the libxmljs package,
305 error not with npm itself.
305 error Tell the author that this fails on your system:
305 error     node-gyp rebuild
305 error You can get their info via:
305 error     npm owner ls libxmljs
305 error There is likely additional logging output above.
306 error System Linux 3.2.0-59-virtual
307 error command "/usr/bin/node" "/usr/bin/npm" "install" "libxmljs"
308 error cwd /home/nemobis/parsoid/tests
309 error node -v v0.10.26
310 error npm -v 1.4.3
311 error code ELIFECYCLE

Full log: http://p.defau.lt/?XhhRQhlxcxpN5kTkleSRqw
It happened with 0.8 but upgrading node to 0.10 didn't help. I read http://grokbase.com/t/gg/nodejs/127j3280ac/installing-libxmljs-in-centos-5-x but it has no working solution

**Comment by MatthewMazaika** [comment]
I saw this exception with a heroku deployment - I commented over https://github.com/TooTallNate/node-gyp/issues/598 with my similar exception - it appears to be a problem with the latest node/npm combination

**Comment by markstos** [comment]
I've also run into an error with `libxmljs` 0.8.0 installing when trying to upgrade from Node 0.10.x to Node 4.2.x. My problem is that can't figure out *which package* has `libxmljs` as a dependency, so  I possibly deal with that package's dependency being out of date. Oddly, when I `grep`  node_modules, there are no hits for `libxmljs` in `package.json` files, yet `npm install` is trying to install it. I tried `npm install --verbose` but it doesn't show where there `libxmljs` dependency originates from that I can see.

**Comment by markstos** [comment]
I figured out what was depending on `libxmljs` through some guess-and-check. I ran `npm install` at least twice and then `checked npm-debug.log`. Since `npm install` had already run once,  on the second run, the first thing it tried to install was failing, which appeared first in the debug log. In my case, this was an old version of `gulp-svgstore`.  I'm not certain if the new version of   `gulp-svgstore` even have `libxmljs` in their dependency tree.
